Source/WebCore/crypto/openssl/CryptoKeyRSAOpenSSL.cpp

```diff
@@ -45,17 +45,9 @@ static size_t getRSAModulusLength(RSA* rsa)
 
 RefPtr<CryptoKeyRSA> CryptoKeyRSA::create(CryptoAlgorithmIdentifier identifier, CryptoAlgorithmIdentifier hash, bool hasHash, const CryptoKeyRSAComponents& keyData, bool extractable, CryptoKeyUsageBitmap usages)
 {
-    CryptoKeyType keyType;
-    switch (keyData.type()) {
-    case CryptoKeyRSAComponents::Type::Public:
+    CryptoKeyType keyType = CryptoKeyType::Private;
```

Comment:
This change doesn't seem meaningful.  If anything, just remove the default

Reply:
removed.